server/liveblog/themes/themes_assets/angular-base/liveblog-embed/filters.js

```diff
@@ -3,8 +3,7 @@
     angular.module('liveblog-embed')
         .filter('prettifyIsoDate', function() {
             return function(input) {
-                var date = fecha.parse(input, 'YYYY-MM-DDTHH:mm:ss+00:00');
-                return fecha.format(date, 'DD/MM/YYYY  HH:mm');
+                return moment(input).format('DD/MM/YYYY  HH:mm');
```

Comment:
finally :100: , please don't forget to remove fecha and add moment in dependencies